src/test/javascript/esutil_test.ts

```diff
@@ -13,13 +13,62 @@
  */
 
 import * as babel from 'babel-types';
+import {parse} from 'babylon';
 import {assert} from 'chai';
 
-import {objectKeyToString} from '../../javascript/esutil';
+import {objectKeyToString, getEventComments} from '../../javascript/esutil';
 
 
 // See analysis_test for tests of generateElementMetadata
 
+suite('getEventComments', () => {
+  test('returns events from a comment', () => {
+    const node = parse(`
+        class Foo {
+          /**
+           * This is an event
+           *
+           * @event event-name
+           * @param {Event} event
```

Comment:
I guess this was already the case, but how can an event have a param? It's not a function. At http://usejsdoc.org/tags-event.html they use @property, which seems to make more sense to me.

Reply:
i meant to ask you about this. its here because in various polymer elements we do this... but i read the jsdoc docs and it isn't mentioned anywhere. it should be a property or type, or something